### DISCUSSION TITLE
support required fields

### DISCUSSION THREAD

**Discussion by jdmann** [tailwindlabs/headlessui#1334] [Ideas]
I don't see a `required` prop on Listbox. This is needed for apps that rely on native form validation. Or will need to build our own logic for this.

**Comment by AnnaYuS** [comment]
It would be nice to have it for the Combobox as well. In Vue version there is a hidden input rendered if we add `name` prop  but `required` doesn't work for a hidden input, so for now to trigger native validation we'll probably end up with adding an "invisible" input
